ADBench/run-all.ps1

```diff
@@ -363,7 +408,13 @@ Class Tool {
             $comparison.CompareFiles($current_jacobian_path, $golden_jacobian_path)
```

Comment:
This is a question not directly about this PR but reading these changes just made me think of it: do we not compare the objective values too?  It looks like we're only comparing the Jacobian.

Reply:
Yes, we compare only the Jacobians. Comparing the objectives themselves would bring some more complexity to run-all, plot-graphs, and JacobianComparisonLib (BA objective is non-trivial), without bringing much value - correct Jacobians mean that the implementation of the objective is off by constant at most, and it generally wasn't produced by the AD tool, so that's not a problem. If the Jacobian is off, checking, whether the the objective is correct would be the first step anyway.

Reply:
> correct Jacobians mean that the implementation of the objective is off by constant at most

Good point